test/e2e/fixture-project/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "dependencies": {
+    "@0no-co/graphqlsp": "file:../../..",
```

Comment:
As we're using pnpm we could leverage workspaces and avoid the need for multiple lockfiles and this directive

Reply:
I added pnpm workspaces, but unfortunately the plugin resolution of TS did not work when I used "@0no-co/graphqlsp": "workspace:*". Probably some symlinking issues

Reply:
In that case might be better to not make a new package for testing and add it at the root

Reply:
Agreed, I'll move `e2e-tests` package stuff to the root.
I think the `fixture-project` should be kept though. It should behave like an isolated environment like an actual project would. 
That won't help with the dependency `"@0no-co/graphqlsp": "file:../../.."` though 😅

Reply:
I got it running with `workspace:*` when adding `hoist-pattern=!@0noco/graphqlsp` to `.npmrc`.